paper-input-container.html

```diff
@@ -594,12 +610,21 @@
           if (label) {
             this.$.labelAndInputContainer.style.position = 'relative';
           }
+          if (invalid) {
+            cls += ' is-invalid';
+          }
         }
       } else {
         if (_inputHasContent) {
           cls += ' label-is-hidden';
         }
+        if (invalid) {
```

Comment:
In my testing, it only moves to "invalid" if it has `auto-validate` and is blurred or it has `validate()` manually called against it. Do you have a context that I should check in on/add a test for in which you think this would occur?